package.json

```diff
@@ -16,22 +16,23 @@
     "body-parser": "1.15.0",
     "connect-flash": "0.1.1",
     "cookie-parser": "1.4.1",
+    "cookie-session": "2.0.0-alpha.1",
     "ejs": "2.4.1",
     "express": "4.13.4",
     "express-session": "1.13.0",
     "font-awesome": "4.6.1",
-    "jquery": "1.12.2",
+    "jquery": "1.12.3",
     "keen-js": "3.4.1",
     "knex": "0.10.0",
     "moment": "2.13.0",
     "moment-timezone": "0.5.4",
     "newrelic": "1.28.0",
     "node-uuid": "1.4.7",
     "nodemailer": "2.4.1",
-    "nodemon": "1.8.1",
+    "nodemon": "1.9.1",
     "passport": "0.3.2",
     "passport-local": "1.0.0",
-    "pg": "4.4.6",
+    "pg": "4.5.1",
```

Comment:
I removed my node_modules folder and ran a fresh npm install. Then npm installed the cookie-session package and npm threw errors explaining that I actually had these versions installed. 

Not quite sure what's going on there, but switching to these version numbers allowed for a successful install. 

-Max

> On Jun 26, 2016, at 3:44 PM, Kuan Butts <notifications@github.com> wrote:
> 
> In package.json:
> 
> >      "passport": "0.3.2",
> >      "passport-local": "1.0.0",
> > -    "pg": "4.4.6",
> > +    "pg": "4.5.1",
> Out of curiosity, how was specific version decided upon?
> 
> ―
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>